esp-wifi/src/ble/btdm.rs

```diff
@@ -80,22 +75,18 @@ extern "C" fn notify_host_recv(data: *mut u8, len: u16) -> i32 {
     trace!("notify_host_recv {:?} {}", data, len);
 
     unsafe {
```

Comment:
Can you change this so that only unsafe operations are wrapped in unsafe, please?